docs/build.md

```diff
@@ -93,16 +93,23 @@ Currently, some tests run on the generated output files of a tool (ex: in order
     1.  Edit the cldr-code project’s Build Path:
         Under “Order and Export”, set the check mark next to “Maven Dependencies”
         so that CLDR makes its dependencies available to the Unicode Tools project.
-3.  Import the unicodetools project into Eclipse. (Using Maven: General > Existing Projects into Workspace)
-4.  Also create the project **and directory** Generated. Various results are
-    deposited there. You need the directory, but the Eclipse project is optional.
-    1.  New... -> Project... -> General/Project
-    2.  Project Name=Generated
-    3.  Uncheck "Use default location" (so that it's not inside your Eclipse workspace)
-    4.  Browse or type a folder path like `Generated` that is a sibling to the top-level `unicodetools` directory
-        1.  Create this folder
-        2.  Create a subfolder BIN
-5.  Project > Clean... > Clean all projects is your friend
+3.  Import the unicodetools project into Eclipse as a Maven project. 
+    1. If your installation of Eclipse does not already include Maven project support, install the [M2Eclipse plugin for Maven support in Eclipse](https://www.eclipse.org/m2e/).
+        1. You can check if the M2Eclipse plugin is installed by looking for the Eclipse "m2" icon in the help box at Help > About (or Eclipse > About Eclipse on macOS).
+        2. If M2Eclipse is not installed, click [here for installation tips](https://www.vogella.com/tutorials/EclipseMaven/article.html#installation-and-configuration-of-maven-for-eclipse).
+    1. Import the Unicode Tools working copy directory as an Eclipse Maven project via
+        1. File > Import ... > Maven > Existing Maven Projects.  Note: if `Maven` and `Existing Maven Projects` don't appear as a top-level category and sub-option in the initial Import screen of the wizard, then the Eclipse plugin for Maven support has not been installed yet, and see above.
+        2. Click Next. In the Root Directory field find the location of the working copy directory. Each pom.xml should be detected and selected in the Projects tree selection widget below. Click Finish to finish importing the Eclipse project.
+4.  Set up a run configuration for building and testing of the entire project using Maven
```

Comment:
This is interesting. This looks plausible, but I haven't done it, and yet have had no problems running/debugging/modifying the tools. Is this optional?

I have been running unit tests via right-clicking on the project and running Maven/JUnit tests.

Reply:
I think that I did run one `mvn build` command or something from the command line when first setting up my workspace, from either the CLDR or Unicode Tools instructions. I don't normally run it though.

I won't belabor this point nor block this PR. But at some point it would be interesting to compare setups and see what's necessary and what's not.